synopsys-task/src/synopsys-task/model/polaris.ts

```diff
@@ -1,13 +1,9 @@
-export enum PolarisAssessmentType {
-  SCA = "SCA",
-  SAST = "SAST",
-}
-
 export interface Polaris {
   polaris: PolarisData;
 }
 
 export interface PolarisData {
+  triage?: string;
```

Comment:
why we are adding ?

Reply:
If ? is added then it is stated as an optional attribute.